com.unity.render-pipelines.high-definition/Runtime/Lighting/Shadow/ScreenSpaceShadowManager.cs

```diff
@@ -895,38 +890,47 @@ bool RenderLightScreenSpaceShadows(HDCamera hdCamera, CommandBuffer cmd)
 
         void EvaluateShadowDebugView(CommandBuffer cmd, HDCamera hdCamera)
         {
-            ComputeShader shadowFilter = m_Asset.renderPipelineRayTracingResources.shadowFilterCS;
-
             // If this is the right debug mode and the index we are asking for is in the range
             HDRenderPipeline hdrp = (RenderPipelineManager.currentPipeline as HDRenderPipeline);
             if (FullScreenDebugMode.ScreenSpaceShadows == hdrp.m_CurrentDebugDisplaySettings.data.fullScreenDebugMode)
             {
-                // Texture dimensions
```

Comment:
This is not the right thing to test here. You should be testing the ray tracing support instead on the pipeline. 

To be fair this is not a good solution either. We should be moving that kernel to a non ray tracing resource to be able to have rasterized directional screen space shadows non depending on ray tracing resources. Alo please change the condition to a "if(..) return" to avoid changing that many lines for a return.